docs/layouts/shortcodes/readfile.html

```diff
@@ -0,0 +1,18 @@
+{{$file := .Get "file"}}
+
+{{- if and (eq (.Get "type") "SECURITY_SCAN") .Site.Params.noSecurityScan -}}
+skip reading security scan for file <em>{{ $file }}</em>
+<br/>
+
+{{- else -}}
+
+{{$file := .Get "file"}}
+{{ if (fileExists $file ) -}}
+{{- if eq (.Get "markdown") "true" -}}
+{{- $file  | readFile | markdownify -}}
+{{- else -}}
+{{ $file  | readFile | safeHTML }}
+{{- end -}}
+{{- end -}}
+
+{{- end -}}
```

Comment:
nit: Should the file end with a new line?

Reply:
It doesn't seem to be strictly necessary
If further commits are needed on this PR I'll include a newline as well